bin/hsw-cli

```diff
@@ -592,93 +592,76 @@ class CLI {
       case 'rescan':
         await this.rescan();
         break;
+      case 'wallets':
+        await this.getWallets();
+        break;
+      case 'mkwallet':
+        await this.createWallet();
+        break;
+      case 'resend':
+        await this.resend();
+        break;
+      case 'backup':
+        await this.backup();
+        break;
+      case 'rpc':
+        await this.rpc();
+        break;
+
       default:
         this.log('Unrecognized command.');
+        this.log('');
         this.log('Commands:');
-        this.log('  $ listen: Listen for events.');
-        this.log('  $ get: View wallet.');
-        this.log('  $ master: View wallet master key.');
-        this.log('  $ shared add [xpubkey]: Add key to wallet.');
-        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
-        this.log('  $ balance: Get wallet balance.');
-        this.log('  $ history: View TX history.');
-        this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
+        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ account create [account-name]: Create account.');
         this.log('  $ account get [account-name]: Get account details.');
+        this.log('  $ account list: List account names.');
         this.log('  $ address: Derive new address.');
+        this.log('  $ balance: Get wallet balance.');
+        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ blocks: List wallet blocks.');
         this.log('  $ change: Derive new change address.');
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
+        this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ master: View wallet master key.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ pending: View pending TXs.');
+        this.log('  $ rescan [height]: Rescan for transactions.');
```

Comment:
`master` and `rescan` require admin, I'm not sure why it wasn't already in that section...

Reply:
Moved